Python/Cryptocurrency_Converter/crypto_converter.py

```diff
@@ -1,110 +1,95 @@
-    except KeyError:
-        return None
+    try: exchange_dict = response_obj.json()["data"]["rates"]
+    except: return None
     return exchange_dict
 
-# This function converts currency amount to cryptocurrency amount
-
-
-def convert_currency_to_crypto(amount, currency, cryptocurrency):
-    """This function converts currency amount to cryptocurrency amount"""
-
-    # checking that the input is correct
-    try:
-        amount = float(amount)
-    except ValueError:
-        return "wrong input!"
-
+def convert_fiat_to_crypto(fvalue, ftype, ctype):
+    """This function converts fiat money to cryptocurrency amount."""
+    available_cryptocurrencies = ['BTC', 'ETH', 'ETC', 'BCH', 'LTC', 'ZEC', 'ZRX']
+    if ctype not in available_cryptocurrencies:
+        print(f"\n{ctype} is not a Cryptocurrency. Try again!")
+        sys.exit(0)
     # get current_price for cryptocurrency using exhange_rates function
-    current_price = exchange_rates(cryptocurrency)[currency]
-
+    try: current_price = exchange_rates(ftype)[ctype]
+    except:
+        print("nNo Information Available. We're Sorry!")
+        sys.exit(0)
     if current_price is None:
-        return "No info available"
-
-    return amount/float(current_price)
-
+        print("\nNo Information Available. We're Sorry!")
+        sys.exit(0)
 
-def convert_crypto_to_currency(amount, cryptocurrency, currency):
-    """This function converts currency amount to cryptocurrency amount"""
+    return float(current_price)*fvalue
 
-    # Just checking the input
-    try:
-        amount = float(amount)
-    except ValueError:
-        return "wrong input!"
 
+def convert_crypto_to_fiat(cvalue, ctype, ftype):
+    """This function converts cryptocurrency amount to fiat money."""
+    available_cryptocurrencies = ['BTC', 'ETH', 'ETC', 'BCH', 'LTC', 'ZEC', 'ZRX']
+    if ftype in available_cryptocurrencies:
+        print(f"\n{ftype} is not a Fiat Money. Try Again!")
+        sys.exit(0)
     # get current_price for cryptocurrency using exhange_rates function
-    current_price = exchange_rates(cryptocurrency)[currency]
-
+    try: current_price = exchange_rates(ctype)[ftype]
+    except:
+        print("\nNo Information Available. We're Sorry!")
+        sys.exit(0)
     if current_price is None:
-        return "No info available"
+        print("\nNo Information Available. We're Sorry!")
+        sys.exit(0)
 
-    return float(current_price)*amount
+    return float(current_price)*cvalue
 
 
 if __name__ == "__main__":
-    STRING = """
-    ################################################################
-
-    examples of fiat currencies :INR,USD,etc.
-    examples of Cryptocurrencies :BTC,ETH,etc.
-    
-    ################################################################
-
-    Choose one of the below options:
-
-    Convert Cryptocurrency to fiat currency (Enter 1)  
-    Convert fiat currency to Cryptocurrency (Enter 2) 
-    Quit (Enter Q)
+    info_msg = """
```

Comment:
Can you extract the whole Input Taking functionality into another UDF? It will be more cleaner to implement and use.